rdkit/Chem/AtomPairs/UnitTestDescriptors.py

```diff
@@ -28,47 +40,59 @@ def testPairsRegression(self):
     atomPairs = cPickle.load(inF, encoding='bytes')
     for i, m in enumerate(self.mols):
       ap = Pairs.GetAtomPairFingerprint(m)
-      #if ap!=atomPairs[i]:
-      #  print Chem.MolToSmiles(m)
-      #  pd=ap.GetNonzeroElements()
-      #  rd=atomPairs[i].GetNonzeroElements()
-      #  for k,v in pd.iteritems():
-      #    if rd.has_key(k):
-      #      if rd[k]!=v: print '>>>1',k,v,rd[k]
-      #    else:
-      #      print '>>>2',k,v
-      #  for k,v in rd.iteritems():
-      #    if pd.has_key(k):
-      #      if pd[k]!=v: print '>>>3',k,v,pd[k]
-      #    else:
-      #      print '>>>4',k,v
-      self.assertTrue(ap == atomPairs[i])
-      self.assertTrue(ap != atomPairs[i - 1])
+      if ap != atomPairs[i]:  # pragma: nocover
```

Comment:
I moved the debugging code (identical for both fingerprints) into a separate function. That makes it more obvious that this is for information only.